README.md

```diff
@@ -245,16 +228,6 @@ functions.cloudEvent('helloCloudEvents', (cloudevent) => {
 });
 ```
 
```

Comment:
The following content is necessary for a cloud event function, maybe we should keep it?
```
To enable the CloudEvent functions, you must list the Functions Framework as a dependency in your `package.json`:
...
```

Reply:
Previous quickstart samples already address the dependency of the framework, so I perfer to omit the dep notice to keep the focus on how to make cloudevent logic works.